ftw/simplelayout/configure.zcml

```diff
@@ -27,8 +27,9 @@
     <include file="lawgiver.zcml" zcml:condition="installed ftw.lawgiver" />
     <include file="resources.zcml" zcml:condition="installed ftw.theming" />
 
-    <include package=".contenttypes" zcml:condition="have ftw.simplelayout:contenttypes" />
-    <include package=".mapblock" zcml:condition="have ftw.simplelayout:contenttypes:mapblock" />
+    <include package=".contenttypes" />
+    <include package=".aliasblock" />
+    <include package=".mapblock" zcml:condition="have ftw.simplelayout:mapblock" />
```

Comment:
is this valid in 1.x?  The contenttypes extra is still explicit there, right?

Reply:
Ahh damn, you are right I thought we already changed this in 1.x 😱